src/psyclone/tests/psyir/transformations/loop_fusion_test.py

```diff
@@ -332,10 +362,12 @@ def test_fuse_correct_bounds(tmpdir, fortran_reader, fortran_writer):
 
 # ----------------------------------------------------------------------------
 def test_fuse_dimension_change(tmpdir, fortran_reader, fortran_writer):
-    '''Test that inconsistent use of dimemsions are detected, e.g.:
+    '''Test that inconsistent use of dimensions are detected, e.g.:
```

Comment:
Strictly, this test now only checks in the case of consistent use of dimensions because, as you say, the rest has been moved.